### DISCUSSION TITLE
multi X-Lite sur x20r

### DISCUSSION THREAD

**Discussion by jeredu12** [FrSkyRC/ETHOS-Feedback-Community#4948] [General]
bonjour je mets le multi x lite je bind un récepteur frsky d8r 2 plus télémètre je veut un retour batterie donc je mets découvert des capteur télémétrie et la radio perd le signal mais le récepteur continue a fonctionner et plus de rssi afficher sauf dans les capteur comment je fait pour que la radio trouve a nouveaux le signal rssi en haut a droite avec la découverte des capteur pour un retour de volt merci

**Comment by bsongis** [comment]
Je pense que c'est un bug sur l'implémentation Multimodule de Ethos. Quelle version de Ethos ?

**Comment by jeredu12** [comment]
merci pour ta raiponce c'est la version 1.5.19. eu et le module ne s'enclanche pas bien dans les fente il faut le mettre de travers

**Comment by bsongis** [comment]
Ce n'est pas corrigé en 1.6.0 ?

**Reply by jeredu12** [reply]
je ne sait pas
